modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -354,6 +395,69 @@ struct IEUnit {
     }
 };
 
+bool IEUnit::InputFrameDesc::is_applicable(const cv::GMetaArg &mm) {
+    switch (mm.index()) {
+        case cv::GMetaArg::index_of<cv::GFrameDesc>():
+            return true;
+        default:
+            return false;
+    }
+}
+
+const IEUnit::InputFrameDesc::description_type &
+IEUnit::InputFrameDesc::get_param(const input_name_type &input) const {
+    auto it = map.find(input);
+    GAPI_Assert(it != map.end() && "No appropriate input is found in InputFrameDesc");
+    return it->second;
+}
+
+void IEUnit::InputFrameDesc::set_param(const input_name_type &input,
+                                       const IE::InputInfo::Ptr& ii) {
+    GAPI_DbgAssert(ii && "IE::InputInfo::Ptr is nullptr");
+    description_type ret;
+    ret.fmt = cv::MediaFormat::NV12;
```

Comment:
In strict words it must be extracted from `face_net.pluginConfig({{"GPU_NV12_TWO_INPUTS", "YES" }});`

Reply:
and checked on `compile` stage

Reply:
What if plugin isn't `GPU`, `GPU_NV12_TWO_INPUTS` won't be passed.
I wouldn't guess color format based on plugin specific options...

Reply:
it moved in `preprocess_frame_impl`: implementation searched it from pluginConfig and apply it if GPU plugin had been  configured before

need to remove explicit setting here